edx_exams/apps/core/signals/handlers.py

```diff
@@ -46,3 +51,17 @@ def listen_for_exam_attempt_rejected(sender, signal, **kwargs):  # pylint: disab
         event_data={'exam_attempt': kwargs['exam_attempt']},
         event_metadata=kwargs['metadata'],
     )
+
```

Comment:
I'd suggest just marking this entire file as `pragma: no-cover` since this pattern of manual sends is temporary

Reply:
excluded this file from coverage by modifying `.coveragerc`